spec/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -207,6 +207,14 @@ function packetCommitmentPath(portIdentifier: Identifier, channelIdentifier: Ide
 
 Absence of the path in the store is equivalent to a zero-bit.
 
+Packet receipt data are stored under the `packetReceiptPath`
+
+```typescript
+function packetReceiptPath(portIdentifier: Identifier, channelIdentifier: Identifier, sequence: uint64): Path {
+    return "acks/ports/{portIdentifier}/channels/{channelIdentifier}/receipts/" + sequence
```

Comment:
the prefix is wrong here, should be `receipts` or smth else

Reply:
Thanks! Fixed in https://github.com/cosmos/ics/commit/46a40a5910fb9e193dccdac48259cee37b34b904 to `receipts`.